### DISCUSSION TITLE
[Pre-release version 22621.4317.67.1: ep_setup.exe downloaded from Assets gets detected as Misleading:Win64/Lodi by Windows Defender

### DISCUSSION THREAD

**Discussion by plofhaan** [valinet/ExplorerPatcher#4064] [General]
This issue can be resolved by adding a file exclusion in Windows Defender. Assuming that the file ep_setup.exe will be downloaded to the Downloads folder of the user that is currently logged in to Windows, running the command below in an elevated command prompt will prevent Windows Defender from detecting the file.

`"%WinDir%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -Command if([Security.Principal.WindowsIdentity]::GetCurrent().Groups -contains 'S-1-5-32-544'){Add-MpPreference -ExclusionPath '%UserProfile%\Downloads\ep_setup.exe' -force;(Get-MpPreference).ExclusionPath}else{'Please run with Administrator privileges.'}`

**Comment by pyrates999** [comment]
Release notes specify for you to do this already.

**Comment by Amrsatrio** [comment]
I'll pull this prerelease.

**Comment by nforystek** [comment]
I think you should say less about the misleading information on the release pages.  Simply that it modifies "explorer.exe" is all you need to say and perhaps accept it as "Malware" because malforming the exe is literally what it sounds like you do.  I think you explain too much.  It is sort of surprising windows doesn't just flat out re-overwrite like it's a system DLL in the protected cache.  Maybe trust goes up if you're an "inverted hey-Zeus."
